### PR TITLE
already good changes

### PR DESCRIPTION
**Pull request by RobinGuillard**
Added the check_imp generation loop
Added the bad epoch classif integration steps
Added the genration of clinical report

### CODE REVIEW

File: Notebooks/Bruxism_detection.ipynb [hunk 1]
```diff
@@ -2,24 +2,23 @@
  "cells": [
```
Comment (lkorczowski) [inline Notebooks/Bruxism_detection.ipynb:2]: <p>Impedence threshold should be there. It should be between #load and #classify.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lkorczowski/Tinnitus-n-Sleep/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (lkorczowski) [inline Notebooks/Bruxism_detection.ipynb:2]: <p>hardcoded parameters for interval the size: 50, 50 which is problematic.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lkorczowski/Tinnitus-n-Sleep/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (lkorczowski) [inline Notebooks/Bruxism_detection.ipynb:2]: <p><code>Impedance_thresholding</code> doesn't use <code>picks_chan</code> parameter. Please remove that in your function and in this code. Also, I advice to rename to </p><p><code>Impedance_thresholding_sliding</code> (and refactoring corresponding <code>test_*.py</code>) to emphasize the fact that it does an epoching.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lkorczowski/Tinnitus-n-Sleep/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (RobinGuillard) [reply Notebooks/Bruxism_detection.ipynb:2]: Ok!

**Pull request merged**